content/docs/dvclive/live/log_artifact.md

```diff
@@ -36,9 +36,9 @@ with Live() as live:
 
 ## Description
 
-Uses `dvc add` to track `path` with DVC, generating a `{path}.dvc` file. When
-combined with [`save_dvc_exp=True`](/doc/dvclive#initialize-dvclive), it will
-ensure that `{path}.dvc` is included in the experiment.
+Uses `dvc add` to [track] `path` with DVC, saving it to the DVC
```

Comment:
tbh, I still feel that we go right away into implementation details. High level (to me) this command first of all creates an entry somewhere (in this case dvc.yaml) with some information about the model - that we now even know that there is a model in the repo. 

`dvc add` can be there or not, we should support other storages and other ways to get data then eventually.

Reply:
> tbh, I still feel that we go right away into implementation details.

I added an intro paragraph to introduce it at a high level.

> High level (to me) this command first of all creates an entry somewhere (in this case dvc.yaml) with some information about the model - that we now even know that there is a model in the repo.
> 
> `dvc add` can be there or not, we should support other storages and other ways to get data then eventually.

I don't really agree here. Every other logger I know uses `log_artifact` to store the artifact data.

Reply:
don't other loggers have any means to log a remote model? 

in our case I think we should support all the ways to store the models. that's why it feels a bit low level and artificial (it can be even a file in Git after all)

Reply:
Yes, we should support other ways to store models (we are discussing in https://github.com/iterative/dvclive/issues/551 and we already have `Live.log_artifact(cache=False) that goes a little bit in this direction), but the expected default behavior for logging artifacts is to save them to the logger's storage so that it can be recovered later. In the intro, I think we should emphasize it as the primary scenario for the method.